internal/cri/server/cni_conf_syncer.go

```diff
@@ -96,18 +98,31 @@ func (syncer *cniNetConfSyncer) syncLoop() error {
 				continue
 			}
 			log.L.Debugf("receiving change event from cni conf dir: %s", event)
-
-			// If the confDir is removed, stop watching.
-			if event.Name == syncer.confDir && (event.Has(fsnotify.Rename) || event.Has(fsnotify.Remove)) {
-				return fmt.Errorf("cni conf dir is removed, stop watching")
-			}
-
 			lerr := syncer.netPlugin.Load(syncer.loadOpts...)
 			if lerr != nil {
 				log.L.WithError(lerr).
 					Errorf("failed to reload cni configuration after receiving fs change event(%s)", event)
 			}
 			syncer.updateLastStatus(lerr)
+			if event.Name == syncer.confDir && (event.Has(fsnotify.Rename) || event.Has(fsnotify.Remove)) {
+				log.L.Warnf("%s is deleted wait it to be recreated", syncer.confDir)
```

Comment:
wording for the warning

Reply:
what warning do you want to show ?  should I add some waning every 5 seconds?